addons/binding/org.openhab.binding.neato/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="neato"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Neato Binding</name>
+    <description>This is the binding for Neato.</description>
+    <author>Patrik Wimnell</author>
+    <config-description>
+        <parameter name="email" type="text">
```

Comment:
According to [the docs](https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-configuration-descriptions), you can set
```
<context>email</context>
```
This is valid, even if the Paper UI might not yet support it - but it might do so in future.